### ISSUE TITLE
doccccccccc

### ISSUE THREAD

**Issue by exploits** [DocMeme/Dotfiles#93]
why did you not respond to Jabber, I added your xmpp :(

**Comment by DocMeme** [comment]
been waiting for weeks to hear from you

**Comment by DocMeme** [comment]
i havent seen any mesages from you on gajim

**Comment by DocMeme** [comment]
Doc@jabber.sk

**Comment by exploits** [comment]
added

**Comment by DocMeme** [comment]
Yeah I know, every time I try to message you it's like you dont get my message or something
![image](https://user-images.githubusercontent.com/19937034/136654375-8c429215-173a-4d4b-878e-e96e7e14c9b3.png)

**Comment by DocMeme** [comment]
just add me on discord when you're able to again Doc#0009

**Comment by DocMeme** [comment]
yeah idk why when i tried to do Doc#0009 it left out the 3 zeros
